scenario_player/runner.py

```diff
@@ -66,36 +65,29 @@ def __init__(
         # Storage for arbitrary data tasks might need to persist
         self.task_storage = defaultdict(dict)
 
-        scenario_name = scenario_file.stem
-        self.base_path = data_path
-        self.base_path.mkdir(exist_ok=True, parents=True)
+        self.definition = ScenarioDefinition(scenario_file, data_path)
 
         log.debug("Local seed", seed=self.local_seed)
 
-        self.data_path = self.base_path.joinpath("scenarios", scenario_name)
-        self.data_path.mkdir(exist_ok=True, parents=True)
-
-        self.definition = ScenarioDefinition(scenario_file, self.data_path)
-        log.debug("Data path", path=self.data_path)
-
-        # Determining the run number requires :attr:`.data_path`
+        # Determining the run number requires :attr:`.yaml`
```

Comment:
What is this comment supposed to tell me?

Reply:
```suggestion
        # Determining the run number requires a scenario definition from `yaml`.
```

Reply:
it's supposed to tell you that you need to load the scenario definition first, before you can determine the run number. It just does a terrible job.